### PR TITLE
find schedds with collector.query() for DUNE global pool

### PR DESCRIPTION
**Pull request by marcmengel**
Quick stab at making dune global pool work again.  Limiting schedd's to ones that match our domain name. 
(This in theory could then work at bnl, as well... maybe?)

### CODE REVIEW

**Comment by marcmengel** [comment]
Okay, so now revisited, it does the whole business of picking a schedd list with one condor collector.query()
Then we aren't contacting the schedd directly, and we don't have problems with the offsite schedd (unless they 
set the IsJobsubLite flag on it, but that's another whole kettle of fish).

File: lib/condor.py [hunk 1]
```diff
@@ -39,61 +39,20 @@ def get_schedd(vargs: Dict[str, Any]) -> classad.ClassAd:
...
     # pylint: disable-next=no-member
-    schedd_classads = coll.locateAll(htcondor.DaemonTypes.Schedd)
-
-    # locateAll gives a list of minimal classads... but
-    # need to directQuery for the full classads to check for
-    # SupportedVOList...
+    devnot = "" if vargs["devserver"] else "!"
+    schedds = coll.query(
+        htcondor.htcondor.AdTypes.Schedd,
+        constraint="isJobsubLite=?=true"
```
Comment (shreyb) [inline lib/condor.py:45]: I don't know if this matters, but I believe`IsJobsubLite` should be capitalized.

Comment (shreyb) [reply lib/condor.py:45]: Certainly the classad is:

```[rexbatch@dunegpcoll01 ~]$ condor_status -schedd -l | grep -i isjobsublite
IsJobsubLite = true
IsJobsubLite = true
IsJobsubLite = false
```

File: lib/condor.py [hunk 2]
```diff
@@ -39,61 +39,20 @@ def get_schedd(vargs: Dict[str, Any]) -> classad.ClassAd:
...
+        " && "
+        f"""STRINGLISTIMEMBER("{vargs['group']}", SupportedVOList)"""
+        " && "
+        "InDownTime != true"
+        " && "
+        f'{devnot} regexp(".*dev.*", Machine)',
```
Comment (shreyb) [inline lib/condor.py:51]: Love it.

**Review by shreyb — CHANGES_REQUESTED** [review]
Just the one bit about capitalization of `IsJobsubLite`

File: lib/condor.py (hunk 1 above)
Comment (marcmengel) [reply lib/condor.py]: Yes.  Missed that, thanks!

**Review by shreyb — APPROVED** [review]
LGTM.  I ran all unit tests, and the following:

1. jobsub_submit -G fermilab --verbose=2  file:///usr/bin/sleep 300
2. jobsub_submit -G fermilab --verbose=2  --devserver file:///usr/bin/sleep 300
3. export _condor_COLLECTOR_HOST=dunegpcoll02.fnal.gov; jobsub_submit -G dune --verbose=2  file:///usr/bin/sleep 300
4. jobsub_submit_dag -G dune --verbose=2  file:///home/sbhat/TestDAG/testdag2.dag
5. unset _condor_COLLECTOR_HOST; jobsub_submit -G dune --verbose=2  file:///usr/bin/sleep 300


All submitted jobs successfully, and to the correct pools.  Will wait on further tests from @marcmengel and @kherner tomorrow morning before we merge this PR into 1.1.2.

**Review by shreyb — APPROVED** [review]

**Review by shreyb — APPROVED** [review]

**Pull request merged**